src/odemis/driver/pwrmccdaq.py

```diff
@@ -403,19 +408,16 @@ def DConfig(self, port_number, bit_mask):
         if bit_mask < 0x00 or bit_mask > 0xff:
             raise ValueError("Bit mask to set is not between 0 and 255")
 
-        if port_number == usb_1208LS.DIO_PORTA:
-            self.port_a_bit_config = [bool(bit_mask & 1 << i) for i in range(8)]
-        elif port_number == usb_1208LS.DIO_PORTB:
-            self.port_b_bit_config = [bool(bit_mask & 1 << i) for i in range(8)]
-        else:
-            raise ValueError()
+        if port_number not in self.port_config:
```

Comment:
In the function's docstring `:param port_number: AUXPORT = 0x10 | Port A = 0x01 | Port B = 0x04`, should port number `AUXPORT` also be handled?